web/react/utils/utils.jsx

```diff
@@ -95,15 +96,20 @@ module.exports.getCookie = function(name) {
   if (parts.length == 2) return parts.pop().split(";").shift();
 }
 
-
 module.exports.notifyMe = function(title, body, channel) {
-  if ("Notification" in window && Notification.permission !== 'denied') {
+    if ("Notification" in window && Notification.permission !== 'denied') {
```

Comment:
Obviously we're going to want to reindent this file as part of a later pull request, but this line is still off since it's indented the same as the following line which is inside the if block.